recipes/mfront/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "mfront" %}
+{% set version = "3.2.1" %}
+{% set sha256 = "12786480524a7fe86889120fb334fa00211dfd44ad5ec71e2279e7adf1ddc807" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: https://github.com/thelfer/tfel/archive/refs/tags/TFEL-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k or osx or win or (python_impl == 'pypy')]
+  detect_binary_files_with_prefix: True
+  run_exports:
+    - {{ pin_subpackage('mfront', max_pin='x.x.x') }}
+    - {{ pin_compatible('python', max_pin='x.x') }}
+
+requirements:
+  build:
+    - gcc 8  # [linux]
+    - gxx 8  # [linux]
+    - gfortran 8  # [linux]
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]    
+    - sysroot_linux-64 =2.17  # [linux]
+  host:
+    - cmake  
+    - make  # [not win]
+    - ninja  # [win]
+    - boost
+    - python {{ python }}
+  run:
+    - python
+    - {{ pin_compatible('boost', max_pin='x.x') }}
+    - libzlib  # [linux]
```

Comment:
I think I added because conda was complaining of a missing run dependency (overlinking error if I am not mistaken), I will try to remove and see what happens. It might also be caused by some "standard" build (using "mamba build" directly to save time instead of "build-locally"), I will see what I can do about this :)

Reply:
Yes I tried and it breaks because of overlinking. I think (might be wrong) that it picks zlib from sysroot then complains because of the missing dependency, maybe I should also add it to the host dependencies ?

Reply:
Added to host dependencies too